chia/rpc/wallet_rpc_api.py

```diff
@@ -2254,7 +2254,7 @@ async def did_recovery_spend(self, request) -> EndpointResult:
                 pubkey = G1Element.from_bytes(hexstr_to_bytes(request["pubkey"]))
             else:
                 assert wallet.did_info.temp_pubkey is not None
-                pubkey = wallet.did_info.temp_pubkey
```

Comment:
@matt-o-how This line is not exercised by any test, and has a (fatal) typo. I don't think it's reasonable to hold up this fix by adding a test. Do you want to vouch for this change?

Reply:
matt approves